projects/cloud_feedbacks/data.md

```diff
@@ -2,8 +2,14 @@
 layout: cloud_feedbacks 
 title: Cloud Feedbacks 
 ---
+
 # Identifying Robust Cloud Feedbacks in Observations and Models 
 
+!({{site.baseurl}}/projects/cloud_feedbacks/binary.svg)
```

Comment:
```suggestion
![binary]({{site.baseurl}}/projects/cloud_feedbacks/binary.svg)
```

Reply:
You need to add the alt text brackets to embed images.

Reply:
is that word going to appear beneath the image? I don't want that...

Reply:
cool! my plan was to make the vertical extent identical to that of the image on the index.md page; is that possible?

Reply:
The alt text only appears if the image file is not found.